fineract-provider/build.gradle

```diff
@@ -725,13 +725,12 @@ springBoot {
     mainClassName = 'org.apache.fineract.ServerApplication'
 }
 
-bootWar {
-    mainClassName = 'org.apache.fineract.ServerApplication'
+bootJar {
     duplicatesStrategy = DuplicatesStrategy.EXCLUDE
 }
 
-bootJar {
-    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+bootWar {
+    enabled = false
```

Comment:
@ptuomola @vidakovic @vorburger Build is successful with bootWar as well but WAR file is not generated in build/libs. 
> Task : bootWar SKIPPED
Flipped enabled of bootWar to true and re-ran the command to generate WAR file. File size is 100mb against v1.4's ~70mb.

Reply:
Let's please discuss this further only on the comments in FINERACT-1203 instead of in 3 places.